test/views/calculation/income/CurrentIncomeViewSpec.scala

```diff
@@ -104,7 +104,7 @@ class CurrentIncomeViewSpec extends CommonPlaySpec with WithCommonFakeApplicatio
       }
 
       s"have the help text ${messages.helpText}" in {
-        doc.body.getElementsByClass("govuk-hint").text shouldBe messages.helpTextShares
+        doc.body.getElementsByClass("govuk-hint").text shouldBe messages.exampleHint
```

Comment:
the message key for the hint should not change - but the original message should have the currency example added to it

Reply:
fixed on [eabcf42](https://github.com/hmrc/cgt-calculator-resident-shares-frontend/pull/184/commits/eabcf42313323762c7b1140a03fb3c5686aef059)